mobify/platform-scaffold
# Ignore generated IM docs

The IM docs are generated into `app/integration-manager/docs` which then are seen by eslint on the next build. They are unrelated to our project and cause a ton of lint errors. We can just ignore them.

 **JIRA**: (link to JIRA ticket)
 **Linked PRs**: (links to corresponding PRs, optional)

## Changes
- Add the IM docs directory to `.eslintignore`

## Todos

## How to test-drive this PR
- Run `npm run im:docs`
- Run `npm run lint` and make sure there are no errors related to code in `app/integration-manager/docs`
